ci-operator/step-registry/openshift-qe/ocp-61589-max-nodes-network-expansion/openshift-qe-ocp-61589-max-nodes-network-expansion-commands.sh

```
@@ -0,0 +1,327 @@
+# Test configuration
+export CLUSTER_NETWORK_ORIGINAL_CIDR="${CLUSTER_NETWORK_CIDR:-10.128.0.0/20}"
+export CLUSTER_NETWORK_EXPANDED_CIDR="${CLUSTER_NETWORK_EXPANDED_CIDR:-10.128.0.0/14}"
+export CLUSTER_NETWORK_HOST_PREFIX="${CLUSTER_NETWORK_HOST_PREFIX:-23}"
+export MAX_NODES_TARGET="${MAX_NODES_TARGET:-520}"
+export EXPECTED_READY_NODES="${EXPECTED_READY_NODES:-510}"
+export TEST_TIMEOUT="${TEST_TIMEOUT:-6h}"
+
+# Function to check cluster health
+check_cluster_health() {
+    echo "Checking cluster operator health..."
+    oc get co --no-headers | while read name _ available progressing degraded _ _; do
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)